dadb/src/main/kotlin/dadb/AdbKeyPair.kt

```diff
@@ -83,7 +83,11 @@ class AdbKeyPair(
         @JvmStatic
         fun read(privateKeyFile: File, publicKeyFile: File): AdbKeyPair {
```

Comment:
Public key file should be nullable and default to null. We should also add the jvmoverloads annotation.